flashinfer/dllm/batch_block_extend.py

```
@@ -0,0 +1,675 @@
+    
+    try:
+        import tvm_ffi
+        results["tvm_ffi_ok"] = True
+        include_path = tvm_ffi.libinfo.find_include_path()
+        device_guard_path = Path(include_path) / "tvm" / "ffi" / "extra" / "cuda" / "device_guard.h"
+        results["device_guard_ok"] = device_guard_path.exists()
+        if not results["device_guard_ok"]:
+            results["issues"].append(f"Missing TVM header: {device_guard_path}")
+    except ImportError:
+        results["issues"].append("tvm_ffi package not installed")
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)